public/components/content-list-item/content-list-item.js

```diff
@@ -62,6 +62,9 @@ function wfContentItemParser(config, statusLabels, sections) {
                 this.live = LIVE_PATH + item.path;
                 this.ophan = OPHAN_PATH + item.path;
             }
+            if (item.contentType === 'media') {
+                this.editor = `${config.mediaAtomMakerViewAtom}${item.editorId}`
```

Comment:
Nope, the snappily named `mediaAtomMakerViewAtom` config item already comes with the slash.